Gemfile

```diff
@@ -99,6 +99,10 @@ group(:linux, :bsd, :mac_os_x, :solaris) do
   gem "ruby-shadow", platform: :ruby
 end
 
+group(:changelog) do
+  gem "github_changelog_generator", git: "https://github.com/tduffield/github-changelog-generator", branch: "adjust-tag-section-mapping"
```

Comment:
For the time being, yes. There is a PR open upstream. Until that is merged or they say they won't accept it, its the best way forward.